### DISCUSSION TITLE
Compatible Devices

### DISCUSSION THREAD

**Discussion by Soltroy** [Skipperro/pajgps-homeassistant#5] [Show and tell]
Just wanted to let you know that you can add
VEHICLE Finder 4G 2.0 to compatible Devices. Batterielevel isn't available, but I guess that's because it's a Hardwired Device.
I'll also keep you updated about the state if you'd like to.

**Comment by skipperro** [comment]
Thanks for the info @Soltroy . 
Since version 0.3.0 this integration should support all of the devices, because it pulls model name and specification (like if device have a battery) directly from the finder-portal API.
